sync/class.jetpack-sync-module-users.php

```diff
@@ -157,6 +157,41 @@ function save_user_cap_handler( $meta_id, $user_id, $meta_key, $capabilities ) {
 		}
 	}
 
+	protected function enqueue_all_ids_as_action( $action_name, $table_name, $id_field, $where_sql ) {
+		global $wpdb;
+
+		if ( ! $where_sql ) {
+			$where_sql = '1 = 1';
+		}
+
+		$items_per_page = 1000;
+		$page           = 1;
+		$chunk_count    = 0;
+		$previous_id    = 0;
+
+		while ( $ids = $wpdb->get_col( "SELECT user_id FROM $wpdb->usermeta WHERE meta_key = '{$wpdb->base_prefix}user_level' AND {$where_sql} AND user_id > {$previous_id} ORDER BY user_id ASC LIMIT {$items_per_page}" ) ) {
```

Comment:
From the [codex](https://codex.wordpress.org/Roles_and_Capabilities#User_Levels):

```
Prior to version 2.0, WordPress used a user User Levels system. This was replaced in version 2.0 with the much improved and more extensible Roles and Capabilities system you see today. To maintain backwards compatibility with plugins that still use the user levels system (although this is very much discouraged), the default Roles in WordPress also include Capabilities that correspond to these levels. User Levels were finally deprecated in version 3.0.
```

This may work for now, but we should figure out a better approach.